src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -69,6 +78,9 @@ public sealed class ColumnCopyingTransformer : OneToOneTransformerBase
         internal const string UserName = "Copy Columns Transform";
         internal const string ShortName = "Copy";
 
+        /// <summary>
+        /// Names of input and ouput column pairs on which the transformation is applied.
```

Comment:
>input and ouput [](start = 21, length = 15)

technically it's `output and input` column

Reply:
Thank you! Will fix


---
In reply to: [253661833](https://github.com/dotnet/machinelearning/pull/2364#discussion_r253661833) [](ancestors = 253661833)